subsys/net/lib/aws_iot/Kconfig

```diff
@@ -78,6 +78,26 @@ config AWS_IOT_TOPIC_DELETE_ACCEPTED_SUBSCRIBE
 config AWS_IOT_TOPIC_DELETE_REJECTED_SUBSCRIBE
 	bool "Subscribe to delete rejected shadow topic, $aws/things/<thing-name>/shadow/delete/rejected"
 
+config AWS_IOT_LAST_WILL
+	bool "This option includes the last will -message and -topic in the initial MQTT handshake"
```

Comment:
Why `-message` and `-topic` and not simply `message`, `topic`? Same below.

Reply:
Well. I guess its not necessary to include the dashes. It was just to emphasize that both "message" and "topic" adheres to "will". But of course, if it looks cleaner omitting the dashes, I can remove them.

Reply:
Had some offline discussion about this. Ill change it to not include the dashes. It looks cleaner.